### DISCUSSION TITLE
Can I use theme colors in this way?

### DISCUSSION THREAD

**Discussion by keene2** [chakra-ui/chakra-ui#8375] [Q&A] [answered]
### Description

color1: {
    100: { light: "#ccc", dark: "#ddd" },
  },

### Link to Reproduction

test

### Steps to reproduce

2. Click on '...'
3. Scroll down to '...'

### Chakra UI Version

"@chakra-ui/react": "^2.6.1",

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment, marked as answer]
That `{ light, dark }` shape is not a token. Put the pair on a semantic token:

```ts
colors: {
  color1: {
    value: { _light: "#ccc", _dark: "#ddd" },
  },
}
```

v2 is maintenance-only. Use `createSystem` + semantic tokens in v3. https://chakra-ui.com/docs/theming/semantic-tokens
